Tribler/Core/Modules/restapi/downloads_endpoint.py

```diff
@@ -169,6 +208,53 @@ def render_DELETE(self, request):
 
         return json.dumps({"removed": True})
 
+    def render_PUT(self, request):
+        """
+        .. http:put:: /download/(string: infohash)
+
+        A PUT request to this endpoint will start a download from a given infohash. Metadata and peers will be fetched
+        from the libtorrent DHT. Various options can be passed:
+        - anon_hops: the number of hops for the anonymous download. 0 hops is equivalent to a plain download
+        - safe_seeding: whether the seeding of the download should be anonymous or not (0 = off, 1 = on)
+        - destination: the download destination path of the torrent
+
+            **Example request**:
+
+                .. sourcecode:: none
+
+                    curl -X PUT http://localhost:8085/download/4344503b7e797ebf31582327a5baae35b11bda01
+                    --data "anon_download=true&safe_seeding=true&destination=/my/dest/on/disk/"
```

Comment:
@whirm ah I forgot to update the doc string here. Is that your only comment regarding this PR? :)